dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -598,6 +592,60 @@ public Response getType(
 		return response;
 	}
 
+	/**
+	 *
+	 * @param req
+	 * @param res
+	 * @param page
+	 * @param perPage
+	 * @param orderBy
+	 * @param direction
+	 * @param form
+	 * @return
+	 */
+	@POST
+	@Path("/allowedtypes")
+	@JSONP
+	@NoCache
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+	public final Response allowedContentTypes(@Context final HttpServletRequest req,
+											  @Context final HttpServletResponse res,
+											  @QueryParam(PaginationUtil.PAGE) final int page,
+											  @QueryParam(PaginationUtil.PER_PAGE) final int perPage,
+											  @DefaultValue("UPPER(name)") @QueryParam(PaginationUtil.ORDER_BY) String orderBy,
+											  @DefaultValue("ASC") @QueryParam(PaginationUtil.DIRECTION) String direction,
+											  final AllowedContentTypesForm form) {
+		if (null == form) {
```

Comment:
Jose I think it would be better to:

Remove the path, and receive everything as a json

{
    page: 1,
    perPage: 2,
  ....
}

Since you already have a post body, lets use it as an only input